src/components/Chart/Chart.tsx

```diff
@@ -80,23 +79,21 @@ export function Chart({ title, type, data }: IChartProps) {
       render={(globalTheme) => {
         const { t } = globalTheme.siteVariables;
         return (
-          <ChartTheme globalTheme={globalTheme}>
-            <React.Suspense fallback={<Loader />}>
-              {data ? (
-                data?.datasets.length || data?.labels?.length ? (
-                  <ChartContainer
-                    title={getText(t.locale, title)}
-                    data={data}
-                    siteVariables={globalTheme.siteVariables}
-                  />
-                ) : (
-                  <ChartEmptyState siteVariables={globalTheme.siteVariables} />
-                )
+          <React.Suspense fallback={<Loader />}>
```

Comment:
There is no need to use `Suspense` here, since this all happens synchronously.

Reply:
Has confirmed. I removed Suspense and confirmed that it renders normally.